servlet/src/main/java/io/undertow/servlet/handlers/security/ServletFormAuthenticationMechanism.java

```diff
@@ -124,9 +126,11 @@ public ServletFormAuthenticationMechanism(FormParserFactory formParserFactory, S
         super(formParserFactory, name, loginPage, errorPage, identityManager);
         this.saveOriginalRequest = true;
     }
-    public ServletFormAuthenticationMechanism(FormParserFactory formParserFactory, String name, String loginPage, String errorPage, IdentityManager identityManager, boolean saveOriginalRequest) {
+    public ServletFormAuthenticationMechanism(FormParserFactory formParserFactory, String name, String loginPage, String errorPage, String defaultPage, boolean overrideInitial, IdentityManager identityManager, boolean saveOriginalRequest) {
```

Comment:
We cant override the constructor, isntead lets add a new one as this class is public and has subclasses (even if tests pass)